package.json

```diff
@@ -32,6 +37,12 @@
     "mysql2": "^1.6.5",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.3",
+    "react-bootstrap": "^1.0.0-beta.5",
```

Comment:
we have react-bootstrap, but you guys rarely use it. Try using it more and you guys can probably reduce a lot lines of code in your components.